hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -414,6 +414,7 @@ private void doCleanObjects() throws InterruptedException {
       }
     } catch (InterruptedException e) {
       e.printStackTrace();
```

Comment:
Unrelated, but this is not the best error handling. At least, we should log it with slf4j rather than print to console.

Reply:
Thanks @jojochuang  for reviewing this.
Is it okay to change `e.printStackTrace();` to `LOG.error("Failed to wait until all Buckets are cleaned", e);`?

Reply:
Or just don’t execute `e.printStackTrace();`?
Which method do you think is better?

Reply:
let's do LOG.error() just like the other cases.

Reply:
Got it.